PICMI_Python/particles.py

```diff
@@ -282,6 +282,77 @@ def __init__(self, density,
         self.handle_init(kw)
 
 
+class PICMI_FoilDistribution(_ClassWithInit):
+    """
+    describes a foil with optional exponential pre-plasma ramps (front lower y/back higher y)
+
+    density: float
+        Physical number density [m^-3]
+
+    y_front : float
```

Comment:
The direction of propagation would be code specific, so don't specify `y`. While it may be the standard definition, it would be helpful to say if this location is the start of the flat part of the foil.
```suggestion
    front : float
```